src/templates/admin/repository/author_article.html

```diff
@@ -259,7 +259,7 @@ <h2>Metrics Summary</h2>
          data-animation-out="slide-out-down">
         <div class="card">
             <div class="card-divider">
-                <h4><i class="fa fa-upload">&nbsp;</i>Galley {{ galley.pk }}</h4>
+                <h4><i class="fa fa-upload">&nbsp;</i>Update Preprint</h4>
```

Comment:
FWIW, I just noticed that `Update {{ request.repository.object_name }}` is used above on line 140